visual_behavior/data_access/loading.py

```diff
@@ -1428,6 +1428,9 @@ def clean_roi_locations_column_labels(roi_locations_dataframe):
     Returns:
         dataframe -- [description]
     """
+    warn_str = gen_depr_str.format('from_lims',
```

Comment:
what's the "warn_str" in this case is?

Reply:
each function has it's own depreciation warning string (warn_str) that's basically a customization of the general depreciation warning string (gen_depr_str). So it gives the general warning "This function is depreciated and will be removed on October 29th 2021. Please use {}.{} instead'   

So it tells you when the function will be removed from the loading.py file and also where you can find the replacement function.